### ISSUE TITLE
docs - getting started - psql: command not found

### ISSUE THREAD

**Issue by francisdb** [digitallyinduced/ihp#1579]
at https://ihp.digitallyinduced.com/Guide/your-first-project.html#the-very-first-time the docs mention to run `make psql`

This however fails with:

```
make psql
/bin/sh: psql: command not found
make: *** [psql] Error 127
```

I guess the user first has to enter a nix shell to get this command working?

```
> nix-shell
[nix-shell:~/mystuff/blog]$ make psql
psql (13.7)
Type "help" for help.

app=#
```

**Comment by mpscholten** [comment]
Could it be that direnv is not working as expected? You can run auto troubleshooting to figure this out https://ihp.digitallyinduced.com/Guide/troubleshooting.html

**Comment by francisdb** [comment]
thanks @mpscholten my direnv was set up for bash in my `~/.zshrc`

```
- eval "$(direnv hook bash)"
+ eval "$(direnv hook zsh)"
```

**Comment by francisdb** [comment]
Looks like this was recently fixed in 5ff5640c34daab4b668437f602ac3fd83cc43aa4
